SydTek.md

```diff
@@ -0,0 +1,228 @@
+| ------: | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|      0. | Rights            | All developed materials will be publicly accessible and public domain. Milestone 2 is research oriented and as such there is no code to test.                                                                                                                                                                                                                                                          |
+|  **1.** | Research Goal     | The SydTek team will interview members of Phala Network, RMRK and Parity Technologies to identify ways users can leverage SBTs and the Social Recovery pallet within the Polkadot and Kusama ecosystems for digital inheritance.         |
+| **1a.** | Specific Coverage | Specific deep dives will propose models and approaches users can leverage to develop proposed solutions of SBT wallets, as well as additional use cases for the Social Recovery pallet.
+
+### Milestone 3 — Peer Review, Journal Publication and Dissemination of Research
+
+- **Estimated Duration:** 1 month
+- **FTE:**  3
+- **Costs:** 9,964 USDC - link to budget details outlined in the Additional Information Section
+
+| Number | Deliverable       | Specification                                                                                                                                                                                                                                                                                                                  |
+| ------: | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|      0. | Rights            | All developed materials will be publicly accessible and public domain. Milestone 3 is research oriented and as such there is no code to test.                                                                                                                                                                                                                                                          |
+|  **1.** | Research Goal     | The SydTek team submit the completed journal article for publication in a peer-reviewed open access journal to ensure the research is available for everyone. The team will then travel to academic institutions across the United States and Canada to disseminate research.         |
```

Comment:
Submission doesn't guarantee publication. What are your plans for if the article is rejected? I just want to make sure it doesn't disappear in the depths of the peer-review apparatus.

Reply:
If the publication is not accepted for publication, we will look to submit and present at blockchain academic conferences, such as the conferences outlined below. Additionally, another option is to publish in arXiv, an open-access repository of research articles maintained by Cornell University. 

![image](https://user-images.githubusercontent.com/77424673/195219842-c890dacb-df4a-482c-8fea-bbd854b64094.png)